robot/Cumulus/resources/CustomizableRollupsPageObject.py

```diff
@@ -43,12 +43,62 @@ def is_setting_present(self, object, name):
             print("crlp setting already exists")
         return isPresent
 
+    def is_filter_present(self, filtername):
+        """ Search for the presence of an active crlp filter setting record already. Return a boolean value accordingly
+		"""
+        isPresent = False
+        filter = npsp_lex_locators['crlps']['filter_group'].format(filtername)
+        self.builtin.log_to_console(filter)
+        list_ele = self.selenium.get_webelements(filter)
+        p_count=len(list_ele)
+        self.builtin.log_to_console(p_count)
+        if p_count == 0:
+            print("Filter Group Not found")
+        else:
+            isPresent = True
+            print("Filter Group Already Exists")
+        return isPresent
+
+    @capture_screenshot_on_error
+    def create_new_filter_setting(self, *args, **kwargs):
+        """Creates a new filter setting by taking in all the filtering criteria(s)
+		"""
+        view_filter_locator = npsp_lex_locators['crlps']['button'].format("View Filter Groups")
+        new_filter_locator = npsp_lex_locators['crlps']['button'].format("New Filter Group")
+        success_toast = npsp_lex_locators['crlps']['success_toast']
+        add_filter_btn = npsp_lex_locators['crlps']['button'].format("Add")
+        self.selenium.wait_until_page_contains_element(view_filter_locator)
+        view_filter_button = self.selenium.get_webelement(view_filter_locator)
```

Comment:
u dont need this line, u can directly call click_element

Reply:
I tried that for some reason this button on this page had an issue hence the workaround